spec/anchor-preview.spec.js

```diff
@@ -40,7 +40,7 @@ describe('Anchor Preview TestCase', function () {
             expect(anchorPreview.showPreview).toHaveBeenCalled();
 
             // link is set in preview
-            expect(anchorPreview.getPreviewElement().querySelector('i').innerHTML).toBe(document.getElementById('test-link').attributes.href.value);
+            expect(anchorPreview.getPreviewElement().querySelector(anchorPreview.previewValueSelector).innerHTML).toBe(document.getElementById('test-link').attributes.href.value);
```

Comment:
I think I would expect the test to fail actually, because they may have broken your behavior which allowed for the link to be right-clicked to be navigated to!

If they're overriding the extension in their own usage of medium-editor, I don't think they'll be running our set of tests against it, but even if they were, they'll need to change the test accordingly to make sure it passes.  If the `<i>` had given us some sort of behavior that was now different, by not referencing the `previewValueSelector` directly in the case, it at least caused you to look at the tests to see what was failing, and I think we'd want someone to do that again if they changed away from an `<a>` tag.  If we use `previewValueSelector` the tests will just pass even though they may have removed your right click to navigate behavior.

My $0.02 anyway